Doc/library/argparse.rst

```diff
@@ -1926,8 +1926,8 @@ FileType objects
       Namespace(out=<_io.TextIOWrapper name='file.txt' mode='w' encoding='UTF-8'>, raw=<_io.FileIO name='raw.dat' mode='wb'>)
 
    FileType objects understand the pseudo-argument ``'-'`` and automatically
-   convert this into ``sys.stdin`` for readable :class:`FileType` objects and
-   ``sys.stdout`` for writable :class:`FileType` objects::
+   convert this into :data:`sys.stdin` for readable :class:`FileType` objects and
+   :data:`sys.stdout`` for writable :class:`FileType` objects::
```

Comment:
```suggestion
   :data:`sys.stdout` for writable :class:`FileType` objects::
```

Reply:
Thanks. I changed it in my repo, but can't push, even after following the hints in the git push message.

Reply:
I'll push it to your fork.

Reply:
It might be that you made a change to the git branch using the GitHub website, and then forgot to `git pull` locally (to update your local branch with the changes you made remotely) before you `git push`ed? That's a common problem everybody runs into :)

Reply:
> I changed it in my repo, but can't push, even after following the hints in the git push message.

If you haven't made other changes that aren't suggestions here, you can also just apply the suggestion(s) you want and commit directly here, which is typically the preferred approach. You go to the `Files` tab, click the `Add to batch` button on the suggestions you want, then click `Commit` with a suitable message.

> I'll push it to your fork.

The other solution would be @smontanaro could simply `git push -f` and then redo the merge with `main`, but he would have to cherry-pick that commit then. The problem now is that if he's also made the change (and possibly others) locally, there may be a merge conflict due to this, which makes things a lot harder to resolve.

> That's a common problem everybody runs into :)

Yup, I see this a lot as well and used to run into it in the past; I avoid it by getting in the habit of always `git pull`ing before making more changes to an existing PR branch (unless I'm absolutely certain there haven't been any) and always rebasing/merging locally